### ISSUE TITLE
Evaluation with model trained on mit gives incorrect results

### ISSUE THREAD

**Issue by anjugopinath** [OpenGVLab/UniFormerV2#24]
Hi,

I tried testing the model trained on mit on a different dataset.
But, I noticed that the video labels in the output .pkl file are the same as that of the labels given in the input file (data_list/mit/test.csv)

Input file (data_list/mit/test.csv):

<img width="217" alt="image" src="https://user-images.githubusercontent.com/4654298/212021823-bdae966d-17f3-451a-acfe-69b48905dd4d.png">

Output file (,pkl file):

<img width="723" alt="image" src="https://user-images.githubusercontent.com/4654298/212022106-778ac3db-7026-4afd-aa38-7fda3bf98a73.png">

As you can see, the labels for the last 11 videos in the input file is "1" and the output video labels are also "1" for the last 11 videos. And "0" for the other videos, again same as the input label.

I don't know what I am doing wrong.

This is the test.sh 

<img width="421" alt="image" src="https://user-images.githubusercontent.com/4654298/212022277-aea57c76-54f1-41b5-a622-6b62680ba8ef.png">

**Comment by Andy1621** [comment]
Thanks for your question. What you have done is right. The video labels mean the ground truth, not the predictions. To achieve the prediction, you should use `torch.from_numpy(res['video_preds']).softmax(-1)`.
I store the labels in the pkl file to calculate the accuracy.
